tests/integration/molecule/module_datastore/playbook.yml

```diff
@@ -0,0 +1,122 @@
+---
+- name: Converge
+  collections:
+    - sensu.sensu_go
+  hosts: all
+  gather_facts: no
+  tasks:
+    - name: Retrieve empty list of external datastores
+      datastore_info:
+        auth: &auth
+          url: http://localhost:8080
+      register: result
+
+    - assert:
+        that:
+          - result.objects == []
+
```

Comment:
Can you add another test here, that ensures validation kicks in, to ensure we cannot create a datastore without a missing `dsn`?